app/controllers/api/messaging.php

```diff
@@ -2640,6 +2641,57 @@
         ]), Response::MODEL_LOG_LIST);
     });
 
+App::get('/v1/messaging/messages/:messageId/targets')
+    ->desc('List message targets')
+    ->groups(['api', 'messaging'])
+    ->label('scope', 'messages.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN, APP_AUTH_TYPE_KEY])
+    ->label('sdk.namespace', 'messaging')
+    ->label('sdk.method', 'listMessageTargets')
```

Comment:
Since the root resource is message, let's just use `listTargets` for the method

Reply:
Updated.